integration/run.sh

```diff
@@ -5,6 +5,8 @@ then
   exit 1
 fi
 
+git config --global --add safe.directory $(pwd)
```

Comment:
this seems to be needed also for some gating runs: https://ci.suse.de/blue/organizations/jenkins/scc-connect-ng-prs/detail/main/57/pipeline/